arangodb-net-standard/Serialization/JsonNetApiClientSerialization.cs

```diff
@@ -39,25 +37,27 @@ public virtual T DeserializeFromStream<T>(Stream stream)
         }
 
         /// <summary>
-        /// Serializes the specified object to a JSON string encoded as UTF-8 bytes,
-        /// following the provided rules for camel case property name and null value handling.
+        /// 
         /// </summary>
-        /// <typeparam name="T">The type of the object to serialize.</typeparam>
-        /// <param name="item">The object to serialize.</param>
-        /// <param name="useCamelCasePropertyNames">Whether property names should be camel cased (camelCase).</param>
-        /// <param name="ignoreNullValues">Whether null values should be ignored.</param>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="item"></param>
+        /// <param name="options"></param>
```

Comment:
I think we should document the fact that `null` options will result in the default options being used.